dockerfiles/QAT/run.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+# NOTE: Some env variables are set in the Dockerfile - those that are
+# intended to be over-rideable.
+QAT_SRC=~/src/QAT
+export GOPATH=~/src/go
+export PATH=${PATH}:/usr/local/go/bin:${GOPATH}/bin
+
+packagerepo=github.com/kata-containers/packaging
+packagerepopath=${GOPATH}/src/${packagerepo}
+osbrepo=github.com/kata-containers/osbuilder
+osbrepopath=${GOPATH}/src/${osbrepo}
+testsrepo=github.com/kata-containers/tests
+testsrepopath=${GOPATH}/src/${testsrepo}
+runtimerepo=github.com/kata-containers/runtime
+runtimerepopath=${GOPATH}/src/${runtimerepo}
+agentrepo=github.com/kata-containers/agent
+agentrepopath=${GOPATH}/src/${agentrepo}
+
+grab_kata_repos()
+{
+	# Check out all the repos we will use now, so we can try and ensure they use the specified branch
+	# Only check out the branch needed, and make it shallow and thus space/bandwidth efficient
+	git clone --single-branch --branch $KATA_VERSION --depth=1 https://${packagerepo} ${packagerepopath}
+	git clone --single-branch --branch $KATA_VERSION --depth=1 https://${osbrepo} ${osbrepopath}
+	git clone --single-branch --branch $KATA_VERSION --depth=1 https://${testsrepo} ${testsrepopath}
+	git clone --single-branch --branch $KATA_VERSION --depth=1 https://${runtimerepo} ${runtimerepopath}
+	git clone --single-branch --branch $KATA_VERSION --depth=1 https://${agentrepo} ${agentrepopath}
+}
+
+configure_kernel()
+{
+	cp /input/qat.conf ${packagerepopath}/kernel/configs/fragments/common/qat.conf
```

Comment:
Seeing this absolute path makes me think it might be worth adding a brief description at the top of the file stating that this is *only* designed to run inside a container.

Reply:
well, it is sat in a folder called `dockerfiles` alongside a `Dockerfile`.... I think it's fairly self explanatory...

Reply:
I think you may underestimate the power of people to make mistakes :smile: It's a script, so somebody is *bound* to try to run it on their system and it will fail because it won't be in a container environment.

If there was a realiable way to detect you were in a docker env, I'd add a test at the top of the script to assert that personally.